crypto/cmp/cmp_ctx.c

```diff
@@ -169,6 +175,13 @@ int OSSL_CMP_CTX_reinit(OSSL_CMP_CTX *ctx)
         && ossl_cmp_ctx_set1_recipNonce(ctx, NULL);
 }
 
+#define OSSL_CMP_ITAVs_free(itavs) \
```

Comment:
Similarly, would these makes sense as functions not macros?

Reply:
Basically the same applies here and for the two further such macros.
Yet I doubt it would be worth spending more effort here because deallocating stacks of these and other types is rather rare.
Just `sk_X509_NAME` may have some significance: 27 occurrences.